pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/route/GetCancerSuggestionsRoute.java

```diff
@@ -44,48 +35,15 @@ public CancerSuggestionResponse handle(Request request, Response response) {
         int limit = StringUtils.isNotBlank(queryLimit) ? Integer.parseInt(queryLimit) : DEFAULT_LIMIT;
         if (StringUtils.isBlank(cancerQuery)) {
             log.info("Cancer query is blank, returning all results");
-            return getUnfilteredCancerSuggestions(cancerQuery, cancerListLanguage, limit);
+            return CancerStore.getInstance().getAllCancers(cancerListLanguage, limit);
         } else {
             if (!Pattern.compile(CANCER_QUERY_REGEX, Pattern.UNICODE_CHARACTER_CLASS).matcher(cancerQuery).find()) {
                 log.warn("Cancer query contains non-alphanumeric characters!");
                 throw ResponseUtil.haltError(response, HttpStatus.SC_BAD_REQUEST,
                         new ApiError(ErrorCodes.MALFORMED_CANCER_QUERY, "Invalid cancer query"));
             }
-            return getCancerSuggestions(cancerQuery, cancerListLanguage, limit);
+            return CancerStore.getInstance().getCancerSuggestions(cancerQuery, cancerListLanguage, limit);
         }
     }
 
-    private CancerSuggestionResponse getUnfilteredCancerSuggestions(String cancerQuery, String language, int limit) {
```

Comment:
Moved these methods to `CancerStore` because 1) I needed to test them explicitly and 2) they seem like general purpose, and shouldn't be confined to a route class.